pyproject.toml

```diff
@@ -102,6 +102,9 @@ filterwarnings = [
 parallel = true
 patch = ["subprocess"]
 source = ["globus_cli"]
+omit = [
+  "globus_cli_flake8.py",
```

Comment:
I have a feeling this shouldn't be in the `src/` tree to begin with, but this is currently a helpful omission!